client/components/Main/ItemEditor/ItemManager.js

```diff
@@ -417,20 +419,25 @@ export class ItemManager {
                 actions.main.post(this.state.initialValues)
             ))
             .then(
-                (updatedItem) => this.setState(!updatedItem ?
-                    // updatedItem === undefined if the user clicks 'Cancel'
-                    // from an 'Ignore/Cancel/Save' dialog
-                    {submitting: false} :
-                    {
-                        submitting: false,
-                        dirty: false,
-                        initialValues: updatedItem,
-                        diff: removeAutosaveFields(
-                            cloneDeep(updatedItem),
-                            true,
-                            true
-                        ),
-                    }, null, true),
+                (updatedItem) => {
+                    this.setState(!updatedItem ?
+                        // updatedItem === undefined if the user clicks 'Cancel'
+                        // from an 'Ignore/Cancel/Save' dialog
+                        {submitting: false} :
+                        {
+                            submitting: false,
+                            dirty: false,
+                            initialValues: updatedItem,
+                            diff: removeAutosaveFields(
+                                cloneDeep(updatedItem),
+                                true,
+                                true
+                            ),
+                        }, null, true);
+                    if (!appConfig.planning_auto_close_popup_editor) {
```

Comment:
What is the scenario here, why is the editor closing?

Reply:
It's mentioned in the ticket requirement:
If the config is set to false, clicking on "post" (or "unpost" for posted events/plannings) should close the popup.
So after updating the Item, we just close the popup.